xwiki-platform-core/xwiki-platform-extension/xwiki-platform-extension-security/xwiki-platform-extension-security-notifications/src/main/java/org/xwiki/extension/security/notifications/NewExtensionSecurityVulnerabilityTargetableEvent.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.extension.security.notifications;
+
+import java.util.Set;
+
+import org.xwiki.eventstream.TargetableEvent;
+
+/**
+ * Targetable event aimed at admins, to inform them of the presence of new security vulnerabilities.
+ *
+ * @version $Id$
+ * @since 15.5RC1
+ */
+public class NewExtensionSecurityVulnerabilityTargetableEvent implements TargetableEvent
+{
+    @Override
+    public boolean matches(Object otherEvent)
+    {
+        return otherEvent instanceof NewExtensionSecurityVulnerabilityTargetableEvent;
+    }
+
+    @Override
+    public Set<String> getTarget()
+    {
+        return Set.of("xwiki:XWiki.XWikiAdminGroup");
```

Comment:
I wouldn't keep it hardcoded in the event itself: I would create the event with a constructor to set the target. We don't know how the event might be used in the future and we might want it to target also other people. So better do it now and it's easy to just target admin group when you're using it now.